libs/coin-modules/coin-stellar/src/bridge/buildOptimisticOperation.ts

```diff
@@ -28,6 +28,7 @@ export async function buildOptimisticOperation(
     transactionSequenceNumber: transactionSequenceNumber?.plus(1).toNumber(),
     extra: {
       ledgerOpType: type,
+      blockTime: new Date(),
```

Comment:
Because of the changes that I have done on the type StellarOperation defined in the file [types/bridge.ts](https://github.com/LedgerHQ/ledger-live/blob/12e9475fabb55f65c4823e63f1c6d0a7b3ac4be0/libs/coin-modules/coin-stellar/src/types/bridge.ts#L105), I needed to update this file or an error of compilation appears. This type is used on the network too. Maybe I need to create a new type only for the network ?